dashboard/app/views/shared/_user_stats.html.haml

```diff
@@ -49,7 +49,7 @@
                 - link += "?user_id=#{user.id}"
               - if @section
                 - link += "&section_id=#{@section.id}"
-              %a.level_link{href: link, class: [status, "level-#{level_id}"], data: {result: result}}
+              %a.level_link{href: link, class: status, data: {result: result}}
```

Comment:
This haml is the view of student progress in the teacher dashboard. level-level_id was also not being used here, so I removed it to avoid confusion.